examples/stable_diffusion_xl/demo/sampling_without_streamlit.py

```diff
@@ -337,5 +376,13 @@ def sample(args):
 if __name__ == "__main__":
     parser = get_parser_sample()
     args, _ = parser.parse_known_args()
-    ms.context.set_context(mode=args.ms_mode, device_target=args.device_target)
+    if args.precision_keep_origin_dtype:
+        ms.context.set_context(
+            mode=args.ms_mode,
+            device_target=args.device_target,
+            ascend_config=dict(precision_mode="must_keep_origin_dtype"),
+        )  # NOTE: Needed for aligning with diffusers
+    else:
+        ms.context.set_context(mode=args.ms_mode, device_target=args.device_target)
```

Comment:
1. is this necessary？

2. Should the comment section be removed as the code base is refer to official implementation

3. modify to:
```
ms.context.set_context(mode=args.ms_mode, device_target=args.device_target)
if args.precision_keep_origin_dtype:
    ms.context.set_context(ascend_config=dict(precision_mode="must_keep_origin_dtype"))
```

Reply:
1. yes if want to minimize the difference btw diffusers and ms inference.
2. removed
3. updated